javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,46 @@
+const width = 10;
+const height = 10;
+const area = width * height;
+
+console.log('value of area: ', area);
+console.log('typeof area: ', typeof area);
+
+const bill = 100;
+const payment = 67;
+const change = bill - payment;
```

Comment:
Your `payment` should be higher than the `bill`, and your `change` operation will need to be swapped

Reply:
Thanks Shawn! See screenshots below!
<img width="519" alt="Screenshot 2023-10-23 at 2 53 45 PM" src="https://github.com/khanhnguyenq/c1023-code-solutions/assets/114459075/a7850668-0796-4231-803e-4047231a9b01">
![Screenshot 2023-10-23 at 2 54 30 PM](https://github.com/khanhnguyenq/c1023-code-solutions/assets/114459075/91ecaadc-11a5-43ea-9d33-441d0bb69195)